internal/handlers/sendBatchMetrics.go

```diff
@@ -0,0 +1,45 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/dcwk/metrics/internal/models"
+)
+
+func (h *Handlers) SendBatchMetrics(metrics map[string]float64, addr string, pollCount *int64) error {
+	path := fmt.Sprintf("http://%s/updates/", addr)
+	metricsList := models.MetricsList{}
+
+	for k, v := range metrics {
```

Comment:
SUGGESTION: it would be better if you move it to another package

Reply:
\+